src/components/tooltip/Tooltip.tsx

```diff
@@ -217,8 +219,27 @@ class Tooltip extends React.Component<TooltipProps, State> {
             stopBubble,
             text,
             theme,
+            intl,
         } = this.props;
 
+        let ariaLabel;
```

Comment:
is undefined a valid ariaLabel?

Reply:
yes it is a valid one, because if is undefined then it will be captured by the line
https://github.com/box/box-ui-elements/pull/2544/files#diff-ecdd760eb0b8901c98cc10e0b0266af590216f4f4619845564b2755d1ef90c55R241

Reply:
Is there a benefit in renaming `ariaLabel` to `childComponentAriaLabel` to keep it clear that it's not the tooltip's `aria-label`?

Reply:
yes!